AppCenter/AppCenter/Internals/Util/MSUtility+StringFormatting.m

```diff
@@ -67,4 +75,17 @@ + (NSString *)transmissionTargetTokenFrom:(NSString *)string {
   }
 }
 
++ (BOOL)isValidAppSecret:(NSString *)appSecretString {
+  if(appSecretString == nil) {
```

Comment:
You can probably merge the two tests, it pretty common to have a nil or empty test.